ui/.env.example

```diff
@@ -5,4 +5,6 @@ NEXT_PUBLIC_BASE_URL=https://localhost:3001
 NEXT_PUBLIC_MEDIA_BUCKET=http://localhost:4566/science-octopus-publishing-images-local
 NEXT_PUBLIC_STAGE=local
 NEXT_PUBLIC_CONTENTFUL_SPACE_ID=contentful-space-id
-NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN=contentful-access-token
\ No newline at end of file
+NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN=contentful-access-token
+NEXT_PUBLIC_MAINTENANCE_MODE=false
+NEXT_PUBLIC_MAINTENANCE_MESSAGE=
```

Comment:
Is the second env var really needed?

Reply:
I double checked this with Tim and he said it could be useful to tell people timeframes + reasons for maintenance in future